idd/Energy+.idd.in

```diff
@@ -84226,6 +84226,70 @@ SetpointManager:ReturnTemperature:HotWater,
        \type object-list
        \object-list ScheduleNames
 
+ SetpointManager:SystemNodeReset:Temperature,
+        \memo This Setpoint Manager is used to place a temperature setpoint on a system node according to the reference (e.g., return) temperature using a reset rule.
+        \memo The temperature setpoint is obtained by retrieving the temperature of the user specified reference system node.
+   A1 , \field Name
+        \required-field
+   A2 , \field Control Variable
+        \required-field
+        \type choice
+        \key Temperature
+        \key MaximumTemperature
+        \key MinimumTemperature
+   N1 , \field Setpoint at Low Reference Temperature
+        \units C
+        \required-field
+   N2 , \field Setppoint at High Reference Temperature
+        \units C
+        \required-field
+   N3 , \field Low Reference Temperature
+        \units C
+        \required-field
+   N4 , \field High Reference Temperature
+        \units C
+        \required-field
+   A3 , \field Reference Node Name
+        \note The name of an HVAC system node that gets referenced.
+        \required-field
+        \type node
+   A4 ; \field Setpoint Node or NodeList Name
+        \note Node(s) at which temperature will be set
+        \required-field
+        \type node
+
+ SetpointManager:SystemNodeReset:Humidity,
+        \memo This Setpoint Manager is used to place a humidity ratio setpoint on a system node according to the reference (e.g., return) humidity ratio using a reset rule.
+        \memo The humidity ratio setpoint is obtained by retrieving the humidity ratio of the user specified reference system node.
+   A1 , \field Name
+        \required-field
+   A2 , \field Control Variable
+        \required-field
+        \type choice
+        \key HumidityRatio
+        \key MaximumHumidityRatio
+        \key MinimumHumidityRatio
+   N1 , \field Setpoint at Low Reference Humidity Ratio
+        \units kgWater/kgDryAir
+        \required-field
+   N2 , \field Setpoint at High Reference Humidity Ratio
+        \units kgWater/kgDryAir
+        \required-field
+   N3 , \field Low Reference Humidity Ratio
+        \units kgWater/kgDryAir
+        \required-field
+   N4 , \field High Reference Humidity Ratio
+        \units kgWater/kgDryAir
+        \required-field
+   A3 , \field Reference Node Name
+        \note The name of an HVAC system node that gets referenced.
+        \required-field
+        \type node
+   A4 ; \field Setpoint Node or NodeList Name
+        \note Node(s) at which humidity ratio will be set
+        \required-field
+        \type node
```

Comment:
Part of me is looking at the similarity between these and wondering if we should collapse them, but I don't think it's appropriate to ask for that change at this point.  And I think set point managers are ripe for a broader cleanup anyway. Carry on...

Reply:
Um, I may have had something to do with splitting this into two objects. :-)

Reply:
It's not a problem, and I get that having them combined may be confusing. Just commenting...